iron-menu-behavior.html

```diff
@@ -117,19 +118,27 @@
       }
     },
 
-    /**
-     * Given a KeyboardEvent, this method will focus the appropriate item in the
-     * menu (if there is a relevant item, and it is possible to focus it).
-     *
-     * @param {KeyboardEvent} event A KeyboardEvent.
-     */
-    _focusWithKeyboardEvent: function(event) {
```

Comment:
From a breaking-changes POV, 'protected' stuff might as well be public since 'extensions' to behaviors are allowed to rely on them ([example](https://github.com/PolymerElements/iron-menu-behavior/blob/e17b3bdb124e42202d50e7c60b0cf02d9bcb201d/iron-menubar-behavior.html#L70)). So we can't actually change this name or what it (conceptually) does. :/

That said, because this method contains all of the behavior for deciding what to focus based on a keyboard event (maybe it's name isn't too bad :smile:), you could move the stuff from `_onKeydown` into here, and `IronMenubarBehavior` could benefit from this without any changes.

p.s. In v2, I'm hoping that we go with unexposed symbols for 'private' references and make the bar for exposing something pretty high. Then we can turn around and say that accessing stuff through symbols we didn't explicitly give you (i.e. you had to rip them off with `Object.getOwnPropertySymbols` or `for in`) is clearly a sign that the property in question is not public API.